src/Templates/ProjectTemplates/VisualBasic/VisualBasicNetStandardClassLibrary/VisualBasicNetStandardClassLibrary.csproj

```diff
@@ -53,7 +53,6 @@
   <ItemGroup>
     <VSTemplate Include="ClassLibrary.vstemplate">
       <SubType>Designer</SubType>
-      <OutputSubPath>.NET Standard</OutputSubPath>
```

Comment:
oh , thought it needs both the directory struture and this. Let me try with just this